apps/frontend/src/modules/projects/hooks/useTheme.ts

```diff
@@ -19,16 +20,17 @@ export type ColorScheme = {
 };
 
 export const useThemes = () => {
```

Comment:
@kaf-lamed-beyt This should be `useTheme` since we are dealing with just a single theme

Reply:
Aha! I see. I'll change it now.